### DISCUSSION TITLE
en/docs/cheat

### DISCUSSION THREAD

**Discussion by giscus** [yuorb/yuorb.github.io#19] [General]
# en/docs/cheat

To be processed, please refer to zsakowitz’ New Ithkuil Single Page Cheat Sheet instead. This enables the inclusion of a list aimed at referencing the most recent Cheat sheets c...

https://yuorb.github.io/en/docs/cheat.html

**Comment by pastequekwsz** [comment]
itkuil

**Reply by ryanlo713** [reply]
Hey! What do you want to discuss about this content?
